image.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Image</title>
+    </head>
+    <body>
+        <!-- here tree image is in same folder i have defined various attributes of img tag -->
```

Comment:
Wrong!!!
actuallly task is that when we click on image it should open second html 'page.

Reply:
ok i made changes